### PR TITLE
Clarify reference server use cases.

### LINKED ISSUE #72: Add an explanation why this anchor server is useful
We should provide a clear explanation about the use cases of this reference implementation in the project's README.

### PR DESCRIPTION
**Pull request by debnil**
This PR clarifies the use cases of the reference server in the README, for both wallets and anchors. It closes #72.

### CODE REVIEW

**Comment by msfeldstein** [comment]
Are we actually going to show SEP 9 and 12?  I thought we're moving away from that

**Comment by msfeldstein** [comment]
Please separate out the part about wallets using it to verify their implementations into a separate line

**Comment by accordeiro** [comment]
> Are we actually going to show SEP 9 and 12? I thought we're moving away from that

Yeah, we were planning to also have a reference for SEP-9 & 12 in our original plan, before they were deprecated. We should remove reference to those SEPs now, and solely focus on SEP-6 and SEP-10.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
...
+Its goal is to provide a community example implementation of [SEP 6](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) (and the related SEP [10](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md)). We hope to make it easier for anchors to integrate with the Stellar network, as they can refer to this sample implementation in their own development. Note that this implementation itself should not be utilized directly (i.e., forked) for handling real amounts of real money.
+Additionally, we want to enable wallets to seamlessly integrate with said anchor. This implementation will provide a reference server for wallets to implement their end of the above SEPs without having to collaborate with an anchor.
```
Comment (msfeldstein) [inline README.md]: you need a second newline here to actually separate the sections

**Pull request merged**